src/compose.ts

```diff
@@ -137,6 +138,9 @@ function normalizeObjectToComposition(
 			case '2.1':
 				version = SchemaVersion.v2_1;
 				break;
+			case '2.4':
+				version = SchemaVersion.v2_4;
+				break;
```

Comment:
How do you plan this to work? What if `c.version` is `'2.2'` or `'2.3'`. Are these versions unacceptable?